src/devices/Bmxx80/ReadResult/Bme280ReadResult.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Units;
+
+namespace Bmxx80.ReadResult
+{
+    /// <summary>
+    /// Measurement results of a Bme280.
+    /// </summary>
+    public struct Bme280ReadResult
+    {
+        /// <summary>
+        /// Collected temperature measurement.
+        /// </summary>
+        public Temperature Temperature;
+
+        /// <summary>
+        /// Collected pressure measurement.
+        /// </summary>
+        public double Pressure;
```

Comment:
You should use the new Pressure unit here (refer to #768), same as it is with Temperature (property above it).